brightray/browser/notification.h

```diff
@@ -26,7 +26,9 @@ class Notification {
                     const std::string& tag,
                     const GURL& icon_url,
                     const SkBitmap& icon,
-                    const bool silent) = 0;
+                    const bool silent,
+                    const bool hasReply,
```

Comment:
There is no need to add `const`, since `bool` is passed by value here, also in C++ we use `snake_case` for variable names:

```c++
bool silent,
bool has_reply,
```